compiler/frontend/beam/src/main/java/edu/snu/nemo/compiler/frontend/beam/NemoPipelineVisitor.java

```diff
@@ -118,6 +118,21 @@ public void visitPrimitiveTransform(final TransformHierarchy.Node beamNode) {
           edge.setProperty(KeyExtractorProperty.of(new BeamKeyExtractor()));
           this.builder.connectVertices(edge);
         });
+
```

Comment:
Add a comment here to note that this is exclusively for Beam nodes with additional tag outputs?